config/default.yml

```diff
@@ -37,6 +37,15 @@ Rails/ActiveRecordAliases:
   VersionAdded: '0.53'
   SafeAutoCorrect: false
 
+Rails/ActiveRecordFind:
+  Description: >-
+                Favor the use of `find` over `where.take!`, `find_by!`, and `find_by_id!` when you
+                need to retrieve a single record by primary key id and raise `ActiveRecord::RecordNotFound`
```

Comment:
Perhaps this could be simplified to "retrieve a single record by primary key when you expect it to exist".